pkg/helm/controller.go

```diff
@@ -283,6 +283,12 @@ func job(chart *helmv1.HelmChart) (*batch.Job, *core.ConfigMap, *core.ConfigMap)
 				Value:    "true",
 				Effect:   "NoSchedule",
 			},
+			{
+				Key:      "CriticalAddonsOnly",
+				Operator: core.TolerationOpEqual,
+				Effect:   "NoExecute",
+				Value:    "true",
```

Comment:
nit: match the field order from the other tolerations
```suggestion
				Value:    "true",
				Effect:   "NoExecute",
```

Reply:
Thanks, addressed.